src/etc/inc/plugins.inc.d/ipsec.inc

```diff
@@ -833,6 +833,8 @@ function ipsec_setup_pinghosts()
     /* get the automatic ping_hosts.sh ready */
     @unlink('/var/db/ipsecpinghosts');
     @touch('/var/db/ipsecpinghosts');
```

Comment:
Okay, I pushed another version now that I better understood what you did mention.
It should reduce I/O in general for the ping hosts script, but also when the function is called.
It will stop re-creating the file for nothing. But the ping hosts script still has to cope with an empty file, since from what I did understand the code will not be called to remove it